hypnotoad/core/equilibrium.py

```diff
@@ -3118,6 +3118,22 @@ def plotPotential(
 
         return axis
 
+    def plotWall(self, ax=None):
```

Comment:
For consistency with `plotPotential` above:
```suggestion
    def plotWall(self, axis=None):
```